pymilo/transporters/transporter.py

```diff
@@ -127,8 +146,19 @@ def transport(self, request, command, is_inner_model=False):
                 model_type = request.type
             for key in data:
                 data[key] = self.deserialize(data, key, model_type)
+            self.reset()
             return
 
         else:
             # TODO error handeling.
             return None
+
+    def reset(self):
+        """
+        Reset internal data structures of the transport object.
+
+        Some Transporters may be stateful and have internal data structures getting filled during transportation.
```

Comment:
No need for this line, I guess.

Reply:
I think it's good to mention that some transporters are stateful.